src/common/models/backend/safe_apps.rs

```diff
@@ -10,6 +10,7 @@ pub struct SafeApp {
     pub description: String,
     pub chain_ids: Vec<u64>,
     pub provider: Option<SafeAppProvider>,
+    pub access_control: SafeAppAccessControlPolicies,
```

Comment:
According to https://github.com/gnosis/safe-config-service/issues/351 this fields is technically `nullable` , right?

This should then be of type `Option<SafeAppAccessControlPolicies>`

Reply:
No, it is not. It's either of these two values

Reply:
Cool, I will just give a heads-up to the team internally, as we would then panic if we try to deploy this to production the CGW before this is available in the prod instance of the config service too :)